Engineering.md

```diff
@@ -1,5 +1,38 @@
 # Engineering Roles
 
+## Junior Engineer
+
+> I am an enthusiastic learner, continually improving my value to an Octopus team.
+
+### I am an egoless, team oriented developer
+
+- Other team members genuinely want me on their team.
+- I happily take responsibility for what I produce.
+- I avoid going dark and I know when to call out for help.
+- I communicate professionally and keep stake holders engaged at all times.
+- I am well aware of all processes of the team and diligently follow them (like maintaining accurate documentation, maintaining tests, and so on).
+- Even when I disagree with something I continue to work enthusiastically, learning from the others in the team.
+- Every venture has mundane tasks, and I work happily and diligently through them.
+- I go out of my way to develop a friendly and professional rapport with my customers and their team members.
+- My contributions to team discussions are net-positive, informative, and helpful.
+
+### I am establishing my ability to write and ship high quality code
+
+- I am learning to write the correct code; code that meets my team's needs.
+- I am learning about complexity and its tradeoffs during software delivery.
+- I am learning to identify good and bad patterns in everyday code. 
+- I am learning how to write automated tests.
+- I am learning about the tools and core building blocks used by our team.
+- I am focussing myself on quality over delivery speed.
+
+### I am establishing my desire to learn and my motivation to improve
+
+- I actively participate in a school to ensure I am maximizing learning opportunities.
```

Comment:
Do we want to be prescriptive here? School might not be the best place to learn for every person in this position

Reply:
Agreed - I don't feel this should necessarily be prescriptive (unless we have something appropriate/general for juniors like #school-octopus or #school-engineering). A school could be a good place to start because mastery is one of their goals, but it also forces them to specialise in a specific area.

Reply:
Thanks I can appreciate that, I'll look at modifying this to be less prescriptive, but ensure there is a sense of "Learning as a primary objective"

Reply:
```suggestion
- I actively seek out learning opportunities that align with my chosen career direction and current skill set.
```